mongodb/doc/MongoImport.md

```diff
@@ -0,0 +1,12 @@
+# MongoDB Import procedure
+
+The MongoDB Import Procedure type is used to import a MongoDB collection into a
```

Comment:
Is it possible to import into an existing dataset?  The description seems to imply that this is possible.